examples/index.html

```diff
@@ -355,6 +355,21 @@ <h2 class="bg-dark">Overlays</h2>
                             </div>
                         </a>
                     </div>
+                    <div class="col-4 p-2">
+                        <a href="custom-bpmn-theme/custom-overlay-default-style/index.html" target="_blank" class="card-link">
+                            <div class="card p-2">
+                                <div class="card-header">
+                                    <div class="card-title">Custom default style for the overlays</div>
+                                    <div class="card-subtitle"></div>
+                                </div>
+                                <div class="card-image">
+                                    <img src="static/img/preview/custom-bpmn-theme/custom-overlay-default-style.png" class="img-responsive" alt="custom colors">
```

Comment:
⚠️ wrong link, examples is now in the overlays directory, so the image is currently not displayed